### DISCUSSION TITLE
How to put a dummy TextInput field in the resources's form that is NOT in database?

### DISCUSSION THREAD

**Discussion by francescverdu** [filamentphp/filament#8911] [Help]
I'm creating a form for a model, and I want to get extra information for special actions after saving the model, but if I put a field name that is not on the database model I get a SQL error  "field XXX not in table". How can I achieve this?

**Comment by emmexx** [comment]
[Field dehydration](https://filamentphp.com/docs/3.x/forms/advanced#field-dehydration)

**Reply by francescverdu** [reply]
I have already tried it, but I always get the SQL error.  
Tried ->dehydrated(false)  and even  ->dehydrated(true), the dehydrated() method seems not to have any efect.

**Reply by emmexx** [reply]
I just tried adding to a form:
```
                        TextInput::make('aName')
                        ->dehydrated(false),
```
and it works, no sql error and aName is not in the "request"
I'm using Filament v3.

**Reply by leandrocfe** [reply]
share your code please

**Comment by francescverdu** [comment]
Oh... now it works fine, I guess what was the problem.

None of my models have $fillables property. Instead, all the models have $guarded=["id"]

Now I've put all model properies in $fillables and tested again. The dummy field is not being sent to SQL and it updates with no errors!

I supose that filament knows the database properties based on $fillables and the names not there are "local" resource data. Or something like this. Am i right?
